bundles/admin/admin-announcements/view/AnnouncementsForm.jsx

```diff
@@ -38,6 +40,11 @@ const AnnouncementsForm = ({controller, key, announcement, bundleKey, index}) =>
     // Should format date value before submit.
     const rangeValue = fieldsValue["range_picker"];
     let content = {...locales};
+    for (const lang in content) {
+      if (content[lang].content && content[lang].content.editorstate) {
+        content[lang].content = stateToHTML(content[lang].content.editorstate.getCurrentContent());
```

Comment:
Could we move `stateToHTML()` handling under `RichEditor`? So we would always get HTML and could input HTML to the editor without needing to know how it goes about it's internal state? Would be a lot nicer to use and we could change the editor implementation easier that way without touching other parts of the code.

Reply:
I think this is good for merging after this change.